app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    command_parts = command.split()
```

Comment:
Code Efficiency: It's a good practice to check if the `mv` command is correct. However, you should also consider the case where the user enters more than one space between the command and the arguments or between the arguments themselves. In your current implementation, such commands will be considered incorrect.